dsl/git_push_test.jmx

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="2.8" jmeter="2.13 r1665067">
```

Comment:
Was this file meant to be committed? The README doesn't mention how to use it and suggests that something similar to this will be generated for you.